components/colors/README.md

```diff
@@ -0,0 +1,25 @@
+# Colors
+
+## d2l-colors
+
+Importing `d2l-colors` will add the color palette to the document and polyfill for IE11 usinf [shadyCSS](https://github.com/webcomponents/shadycss). Run the demo page to see the full palette.
```

Comment:
Technically not called `d2l-colors` anymore.

Reply:
Good point - will update.